README.md

```diff
@@ -76,7 +76,7 @@ client.on('ready', () => {
 
 client.on('message', msg => {
   if (msg.content === 'ping') {
-    msg.reply('pong');
+    msg.channel.send('pong');
```

Comment:
Why have a lot of `msg.reply` examples been removed, when the method is still available?

Reply:
Likely caused by the merge of my removal, which has been incorporated into this PR. However I also don't feel like we should use inline replies as the standard way to do command replies either, making the replacement still valid imo

Reply:
Doesn't mean remove everything... especially from typings

Reply:
The removal of the typings is an oversight, as I was originally planning to name this method something different to maintain it as minor rather than major. Will add them back